From e47c86ee3434b6d2e7a565115f5476ca6e29d1a4 Mon Sep 17 00:00:00 2001
From: Sandor Yu <R01008@freescale.com>
Date: Wed, 5 Sep 2012 11:17:32 +0800
Subject: [PATCH 2009/2463] ENGR00221854 HDMI: suspend/resume hdmi_phy fail to
 lock.

HDMI PHY init function will been called four times during system resume.
The first call before pixel clock enable, so it will print
PHY PLL unlock message, but the PHY PLL will locked in the next
three times called. It will not affect HDMI PHY function.

Change message print function dev_err to dev_dbg.

Signed-off-by: Sandor Yu <R01008@freescale.com>
---
 drivers/video/mxc_hdmi.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/video/mxc_hdmi.c b/drivers/video/mxc_hdmi.c
index 1832a59..e9c0547 100644
--- a/drivers/video/mxc_hdmi.c
+++ b/drivers/video/mxc_hdmi.c
@@ -1070,7 +1070,7 @@ static int hdmi_phy_configure(struct mxc_hdmi *hdmi, unsigned char pRep,
 	while (val == 0) {
 		udelay(1000);
 		if (msec-- == 0) {
-			dev_err(&hdmi->pdev->dev, "PHY PLL not locked\n");
+			dev_dbg(&hdmi->pdev->dev, "PHY PLL not locked\n");
 			return false;
 		}
 		val = hdmi_readb(HDMI_PHY_STAT0) & HDMI_PHY_TX_PHY_LOCK;
-- 
1.7.7.4

